File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        i =0
        zero = 0
        two = len(nums)-1
        while i<=two:
            if nums[i] == 0:
                temp = nums[zero]
                nums[zero] = nums[i]
                nums[i] = temp
                zero +=1
                i += 1
            elif nums[i] == 2:
                temp = nums[two]
                nums[two] = nums[i] 
                nums[i] = temp
                two -=1
            else:
                i +=1
        return

        

File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int i =0;
        int zero = 0;
        int two = nums.size()-1;
        while(i<=two){
            if(nums[i] == 0){
                int temp = nums[zero];
                nums[zero] = nums[i];
                nums[i] = temp;
                zero++;
                i++;
            }
            else if(nums[i] == 2){
                int temp = nums[two];
                nums[two] = nums[i];
                nums[i] = temp;
                two--;
            }
            else i++;
        }
        return;
    }
};